### PR TITLE
RS: Add REST API references for /cluster/check and /nodes/check

### PR DESCRIPTION
**Pull request by rrelledge**
DOC-3668

### CODE REVIEW

File: content/operate/rs/references/rest-api/requests/cluster/check.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+| Check name | Description |
+|-----------|-------------|
+| bootstrap_status | Verifies the local node's bootstrap process completed without errors. |
+| services | Verifies all Redis Enterprise Software services are running. |
+| port_range | Verifies the [`ip_local_port_range`](https://www.kernel.org/doc/html/latest/networking/ip-sysctl.html) doesn't conflict with the ports Redis Enterprise might assign to shards. |
+| pidfiles | Verifies all active local shards have PID files. |
+| capabilities | Verifies all binaries have the proper capability bits. |
+| existing_sockets | Verifies sockets exist for all processes that require them. |
+| host_settings | Verifies the following:<br />• Linux `overcommit_memory` setting is 1.<br />•`transparent_hugepage` is disabled.<br />• Socket maximum connections setting `somaxconn` is 1024. |
```
Comment (dwdougherty) [inline content/operate/rs/references/rest-api/requests/cluster/check.md:33]: Subtle change: adds space after 'dot'; looks like it's there for the others.

```suggestion
| host_settings | Verifies the following:<br />• Linux `overcommit_memory` setting is 1.<br />• `transparent_hugepage` is disabled.<br />• Socket maximum connections setting `somaxconn` is 1024. |
```

File: content/operate/rs/references/rest-api/requests/cluster/check.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+### Response {#get-response} 
+
+Returns JSON array with results from all nodes.
+
+When errors occur, the server returns a JSON object with `result: false` and an `error` field that provides additional information for each node that had an error. If an error occurs during a check, the `error` field only includes a message for the first check that fails on each node.
+
+Possible `error` messages:
+
+- "bootstrap request to cnm_http failed,resp_code: ...,resp_content: ..."
```
Comment (dwdougherty) [inline content/operate/rs/references/rest-api/requests/cluster/check.md:65]: Should there be a space before resp_code?

File: content/operate/rs/references/rest-api/requests/cluster/check.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    }
+  ]
+}
+```
+
+### Status codes {#get-status-codes} 
+
+| Code | Description |
+|------|-------------|
+| [200 OK](http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.2.1) | No error |
```
Comment (dwdougherty) [inline content/operate/rs/references/rest-api/requests/cluster/check.md:97]: Is there a newer URL to use here? Seems the current link points to an out-of-date resource.

File: content/operate/rs/references/rest-api/requests/nodes/check.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
+---
```
Comment (dwdougherty) [inline content/operate/rs/references/rest-api/requests/nodes/check.md:1]: Same comments as before.

**Review by dwdougherty — APPROVED** [review]
A few comments. I'll approve anyway so I don't hold you up.

File: content/operate/rs/references/rest-api/requests/cluster/check.md (hunk 2 above)
Comment (rrelledge) [reply content/operate/rs/references/rest-api/requests/cluster/check.md:65]: I double-checked the source code in GitHub and confirmed there is no space between the comma and `resp_content` in the error message.

File: content/operate/rs/references/rest-api/requests/cluster/check.md (hunk 3 above)
Comment (rrelledge) [reply content/operate/rs/references/rest-api/requests/cluster/check.md]: Fixed

File: content/operate/rs/references/rest-api/requests/cluster/check.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+| Key | Value | Description |
+|-----|-------|-------------|
+| Host | cnm.cluster.fqdn | Domain name |
+| Accept | application/json | Accepted media type |
+
+
+### Response {#get-response} 
+
+Returns JSON array with results from all nodes.
```
Comment (andy-stark-redis) [inline content/operate/rs/references/rest-api/requests/cluster/check.md:59]:
```suggestion
Returns a JSON array with results from all nodes.
```

**Review by andy-stark-redis — APPROVED** [review]
Minor suggestion but otherwise the language looks good. Just a thought for a future task - do you think the tables would look better with the headings left-aligned? The other pages in this section have the same table alignment, so no point fixing it just for this page.

**Pull request merged**